piccolo/apps/playground/commands/run.py

```diff
@@ -253,4 +256,14 @@ def run(
 
     from IPython.core.interactiveshell import _asyncio_runner  # type: ignore
 
-    IPython.embed(using=_asyncio_runner)
+    if ipython_profile:
+        print("Using user ipython profile")
+        print(
+            "More info: https://ipython.org/ipython-doc/3/config/intro.html#"
```

Comment:
I'd change this link to this:

https://ipython.readthedocs.io/en/stable/config/intro.html

As it will point to the latest version of the docs.